src/test/functional_tests.rs

```diff
@@ -54,8 +55,8 @@ fn channel_full_cycle() {
 	node_b.sync_wallets().unwrap();
 
 	let node_a_balance = node_a.on_chain_balance().unwrap();
-	assert!(node_a_balance.get_spendable() < 50000);
-	assert!(node_a_balance.get_spendable() > 40000);
+	assert!(node_a_balance.get_spendable() < 20000);
+	assert!(node_a_balance.get_spendable() > 15000);
```

Comment:
Why not `10000`? Any reason why we don't assert exact values?

Reply:
Yeah, I'd like to at least make the code clearer that this happening by writing the asserts as the arithmetic of variables with meaningful names.